vignettes/programming_strategy.Rmd

```diff
@@ -224,9 +224,9 @@ Arguments which only expect one value or variable name must be a singular versio
 | `start_dtc`      | (Partial) start date/datetime in ISO 8601 format.                                                                  |
 | `dtc`            | (Partial) date/datetime in ISO 8601 format.                                                                        |
 | `date`           | Date of an event / interval. Expects a date object.                                                                |
-| `set_values_to`  | List of variable name-value pairs. Use `process_set_values_to()` for processing the value and providing user friendly error messages. |
 | `subject_keys`   | Variables to uniquely identify a subject, defaults to `exprs(STUDYID, USUBJID)`. In function formals, use `subject_keys = get_admiral_option("subject_keys")`                                     |
-| `keep_source_vars` | Specifies which variables from the selected observations should be kept. The default of the argument should be `everything()`.                                          |
+| `set_values_to`  | List of variable name-value pairs. Use `process_set_values_to()` for processing the value and providing user friendly error messages. |
+| `keep_source_vars` | Specifies which variables from the selected observations should be kept. The default of the argument should be `everything()`. The primary difference between `set_values_to` and `keep_source_vars` is that `keep_source_vars` only selects and retains the variables from a source dataset, so e.g. `keep_source_vars = exprs(DOMAIN)` would join + keep the `DOMAIN` variable, whereas `set_values_to` can make renaming and inline function changes such as `set_values_to = exprs(LALVDOM = DOMAIN)`                       |
```

Comment:
Could you change `everything()` to `exprs(everything())`? Then it is in line with our implementation.

Reply:
Fixed!